### PR TITLE
Patch release 3.0.2 for fixing BUSCO, CONCOCT and Docker container errors

### PR DESCRIPTION
**Pull request by jfy133**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/mag/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/mag _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,29 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 3.0.2 [2024-06-28]
```
Comment (jfy133) [inline CHANGELOG.md:6]:
```suggestion
## 3.0.2 [2024-07-01]
```

**Review by prototaxites — APPROVED** [review]
LGTM! 👍

**Review by mashehu — APPROVED** [review]

**Review by christopher-hakkaart — APPROVED** [review]
Looks good 👍

**Comment by jfy133** [comment]
Thank you @prototaxites @mashehu @christopher-hakkaart !

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,29 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 3.0.2 [2024-07-01]
```
Comment (jfy133) [inline CHANGELOG.md:6]:
```suggestion
## 3.0.2 [2024-07-04]
```

**Pull request merged**